resources/vault_install_binary.rb

```diff
@@ -23,11 +23,13 @@
 default_action :install
 
 action :install do
-  directory new_resource.extract_to do
-    mode '0755'
-    owner new_resource.user
-    group new_resource.group
-    recursive true
+  [new_resource.extract_to, ::File.join(new_resource.extract_to, new_resource.version)].each do |dir|
+    directory dir do
+      mode '0755'
+      owner new_resource.user
+      group new_resource.group
+      recursive true
+    end
```

Comment:
While the poise extract does create the files and set the owner/group, it does not catch the mode which made one of our tests fail. How do you think that should be handled?